src/firestore/provisioning.ts

```diff
@@ -0,0 +1,54 @@
+import { firestoreOrigin } from "../api";
+import { Client } from "../apiv2";
+import { FirebaseError } from "../error";
+
+// todo add enums for possible string values
```

Comment:
Consider a string literal union instead of an enum: https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types

Reply:
Also, TODOs should have owners or bugs attached:
`// TODO(VicVer09): ... `